hzdynamics/parameters_vel_1.txt
import numpy as np

a_rightS = np.array([2.3350, 0.5286, 3.1350, 0.1819, 2.4452, 0.4914, 
3.1404, 0.3618, 2.7127, 0.2360, 2.8894, 0.9955, 
2.9517, 0.0596, 2.4470, 1.0882, 3.0557, 0.0315, 
2.3055, 0.8447, 3.1350, 0.1819, 2.3349, 0.5286])

a_leftS = np.array([3.1350, 0.1819, 2.3350, 0.5286, 3.1404, 0.3618, 
2.4452, 0.4914, 2.8894, 0.9955, 2.7127, 0.2360, 
2.4470, 1.0882, 2.9517, 0.0596, 2.3055, 0.8447, 
3.0557, 0.0315, 2.3349, 0.5286, 3.1350, 0.1819])

p = np.array([0.2817, -0.2024])

position = np.array([[-0.2024, 0.7452, 0.2771, 2.3350, 0.5286, 3.1350, 0.1819],
[-0.1776, 0.7531, 0.2860, 2.3671, 0.5138, 3.1297, 0.2386],
[-0.1538, 0.7606, 0.2923, 2.4044, 0.4908, 3.1127, 0.3086],
[-0.1308, 0.7676, 0.2965, 2.4451, 0.4625, 3.0860, 0.3848],
[-0.1085, 0.7740, 0.2989, 2.4881, 0.4306, 3.0513, 0.4626],
[-0.0868, 0.7795, 0.3000, 2.5324, 0.3967, 3.0102, 0.5386],
[-0.0655, 0.7842, 0.3000, 2.5774, 0.3618, 2.9640, 0.6106],
[-0.0445, 0.7881, 0.2992, 2.6227, 0.3267, 2.9138, 0.6767],
[-0.0236, 0.7911, 0.2979, 2.6679, 0.2920, 2.8604, 0.7358],
[-0.0026, 0.7933, 0.2963, 2.7128, 0.2584, 2.8047, 0.7870],
[0.0185, 0.7947, 0.2945, 2.7572, 0.2262, 2.7472, 0.8294],
[0.0399, 0.7952, 0.2927, 2.8011, 0.1959, 2.6888, 0.8622],
[0.0619, 0.7948, 0.2910, 2.8442, 0.1682, 2.6302, 0.8843],
[0.0847, 0.7934, 0.2895, 2.8865, 0.1437, 2.5722, 0.8947],
[0.1084, 0.7911, 0.2884, 2.9277, 0.1233, 2.5161, 0.8920],
[0.1332, 0.7876, 0.2875, 2.9675, 0.1082, 2.4635, 0.8747],
[0.1594, 0.7829, 0.2869, 3.0055, 0.1000, 2.4163, 0.8415],
[0.1871, 0.7768, 0.2864, 3.0414, 0.1005, 2.3769, 0.7908],
[0.2166, 0.7688, 0.2853, 3.0749, 0.1123, 2.3483, 0.7218],
[0.2481, 0.7586, 0.2828, 3.1058, 0.1381, 2.3334, 0.6340],
[0.2817, 0.7452, 0.2771, 3.1350, 0.1819, 2.3350, 0.5286]])

velocity = np.array([[0.9450, 0.2960, 0.3870, 1.0761, -0.3633, 0.0529, 1.7556],
[0.9028, 0.2882, 0.2802, 1.3023, -0.7225, -0.4285, 2.4068],
[0.8685, 0.2714, 0.1920, 1.4577, -0.9669, -0.8257, 2.7508],
[0.8402, 0.2482, 0.1200, 1.5620, -1.1296, -1.1517, 2.8880],
[0.8173, 0.2208, 0.0626, 1.6275, -1.2305, -1.4165, 2.8763],
[0.7992, 0.1909, 0.0180, 1.6646, -1.2857, -1.6295, 2.7622],
[0.7862, 0.1599, -0.0156, 1.6811, -1.3059, -1.7985, 2.5744],
[0.7784, 0.1286, -0.0397, 1.6832, -1.2997, -1.9311, 2.3352],
[0.7761, 0.0972, -0.0557, 1.6756, -1.2722, -2.0328, 2.0564],
[0.7799, 0.0658, -0.0648, 1.6615, -1.2268, -2.1081, 1.7455],
[0.7901, 0.0342, -0.0679, 1.6425, -1.1638, -2.1581, 1.4030],
[0.8070, 0.0019, -0.0658, 1.6189, -1.0817, -2.1819, 1.0270],
[0.8310, -0.0319, -0.0593, 1.5898, -0.9762, -2.1745, 0.6117],
[0.8623, -0.0677, -0.0492, 1.5533, -0.8412, -2.1284, 0.1507],
[0.9009, -0.1068, -0.0372, 1.5070, -0.6674, -2.0315, -0.3618],
[0.9471, -0.1504, -0.0257, 1.4482, -0.4440, -1.8695, -0.9301],
[1.0010, -0.2006, -0.0198, 1.3758, -0.1566, -1.6255, -1.5522],
[1.0626, -0.2603, -0.0267, 1.2904, 0.2111, -1.2826, -2.2211],
[1.1321, -0.3349, -0.0602, 1.1985, 0.6793, -0.8293, -2.9149],
[1.2093, -0.4321, -0.1389, 1.1121, 1.2702, -0.2618, -3.6012],
[1.2929, -0.5668, -0.2939, 1.0596, 2.0079, 0.3913, -4.2187]])


hzdynamics/parameters_vel_0.5.txt
a_rightS = np.array([8.7324, 2.4956, 9.6066, 1.6713, -93.1458, -35.3486, 
-99.9992, -19.7280, 86.8247, 31.3159, 93.6746, 19.3897, 
-26.4069, -11.2585, -29.1269, -5.1783, 4.5299, 0.7725, 
4.1839, 0.7467, 3.9968, 0.9334, 3.6856, 0.2763])

a_leftS = np.array([4.3111, 0.7323, 3.8308, 0.7187, 99.9046, 19.4248, 
92.7007, 33.8664, -91.4680, -16.8641, -85.2398, -32.8513, 
37.4937, 7.8189, 35.5215, 11.8593, 1.2146, 0.1866, 
1.7585, -0.2782, 1.2946, -0.2103, 1.7679, 0.1228])

p = np.array([0.2000, -0.2000])

position = np.array([[-0.1938, 0.7690, 0.0513, 2.7047, 0.2743, 3.1148, 0.4455],
[-0.1883, 0.7724, 0.2025, 2.6499, 0.2464, 3.0135, 0.4593],
[-0.1759, 0.7760, 0.2495, 2.5999, 0.1964, 2.9649, 0.5299],
[-0.1611, 0.7792, 0.2909, 2.5948, 0.1599, 2.9420, 0.6147],
[-0.1452, 0.7817, 0.2990, 2.5970, 0.1289, 2.9399, 0.7045],
[-0.1239, 0.7844, 0.2530, 2.6368, 0.1113, 2.9849, 0.8184],
[-0.0949, 0.7877, 0.1911, 2.7848, 0.1295, 3.0889, 0.9569],
[-0.0675, 0.7906, 0.1180, 2.9158, 0.1397, 3.1588, 1.0475],
[-0.0462, 0.7924, 0.0575, 2.9884, 0.1325, 3.1643, 1.0689],
[-0.0268, 0.7937, 0.0112, 3.0523, 0.1285, 3.1499, 1.0611],
[-0.0125, 0.7945, -0.0077, 3.0987, 0.1280, 3.1201, 1.0361],
[-0.0026, 0.7951, -0.0040, 3.1137, 0.1234, 3.0796, 1.0063],
[0.0064, 0.7955, 0.0009, 3.1177, 0.1180, 3.0368, 0.9715],
[0.0194, 0.7955, -0.0324, 3.1284, 0.1125, 3.0066, 0.9330],
[0.0361, 0.7949, -0.0403, 3.2004, 0.1375, 2.9898, 0.8937],
[0.0515, 0.7936, -0.0445, 3.2446, 0.1756, 2.9655, 0.8279],
[0.0663, 0.7905, -0.0695, 3.2336, 0.2211, 2.9322, 0.7249],
[0.0843, 0.7844, -0.0976, 3.2324, 0.2932, 2.9060, 0.5980],
[0.1030, 0.7767, -0.0879, 3.2557, 0.3824, 2.8774, 0.4680],
[0.1168, 0.7717, -0.0502, 3.2187, 0.4381, 2.8202, 0.3571],
[0.1220, 0.7702, 0.0547, 3.1144, 0.4437, 2.7077, 0.2746],
])

velocity = np.array([[0.0015, 0.0740, 4.1754, -3.4967, -1.3033, -3.7623, -0.7381],
[0.2681, 0.0961, 2.3015, -1.3867, -1.1046, -1.8248, 1.3215],
[0.3682, 0.0910, 1.4284, -0.4384, -1.0260, -0.9449, 2.2530],
[0.4131, 0.0761, 0.8059, 0.1239, -0.8013, -0.3372, 2.3813],
[0.4548, 0.0638, 0.2321, 0.6340, -0.5754, 0.1863, 2.3781],
[0.6861, 0.0809, -1.5904, 2.4612, -0.0140, 2.1490, 3.4294],
[0.8746, 0.1002, -3.0522, 4.0128, 0.2999, 3.2292, 3.8477],
[0.6328, 0.0602, -1.6720, 2.4956, -0.0305, 0.7457, 1.2489],
[0.5528, 0.0437, -1.2760, 2.0122, -0.0590, -0.1966, 0.0973],
[0.4743, 0.0272, -0.9284, 1.5536, -0.0335, -0.5816, -0.4937],
[0.3195, 0.0217, -0.1425, 0.6267, -0.1417, -0.9499, -0.6734],
[0.2536, 0.0169, 0.2145, 0.1889, -0.1401, -1.1574, -0.8517],
[0.2795, 0.0094, 0.0993, 0.2821, -0.0295, -1.1240, -1.0068],
[0.4317, -0.0069, -0.7375, 1.1695, 0.2467, -0.5619, -1.0215],
[0.4702, -0.0142, -0.8902, 1.2757, 0.4485, -0.4205, -1.1758],
[0.4010, -0.0534, -0.3344, 0.3045, 1.1251, -0.8237, -2.3056],
[0.4392, -0.1052, -0.3081, 0.0355, 1.7302, -0.9191, -3.1541],
[0.5327, -0.1990, -0.6031, 0.2373, 2.1965, -0.6073, -3.5167],
[0.4788, -0.1951, 0.1097, -0.3888, 1.8911, -1.0259, -3.3032],
[0.2814, -0.1014, 1.8020, -1.8682, 0.9332, -2.1358, -2.6076],
[0.0263, 0.1081, 3.9969, -3.6266, -0.7138, -3.9951, -1.7622],
])



hzrl/hzd/params.py
import numpy as np

position = np.array([[-0.2000, 0.7546, 0.1675, 2.4948, 0.4405, 3.1894, 0.2132],
[-0.1484, 0.7460, 0.1734, 2.4568, 0.6307, 2.9492, 0.6271],
[-0.1299, 0.7518, 0.1767, 2.4873, 0.6133, 2.9434, 0.6740],
[-0.1121, 0.7567, 0.1794, 2.5177, 0.5954, 2.9297, 0.7256]])

velocity = np.array([[0.7743, 0.2891, 0.3796, 1.1377, -0.9273, -0.1285, 1.6298],
                     [0, 0, 0, 0, 0, 0, 0],
                     [0, 0, 0, 0, 0, 0, 0]])



a_rightS = np.array([2.3350, 0.5286, 3.1350, 0.1819, 2.4452, 0.4914, 
3.1404, 0.3618, 2.7127, 0.2360, 2.8894, 0.9955, 
2.9517, 0.0596, 2.4470, 1.0882, 3.0557, 0.0315, 
2.3055, 0.8447, 3.1350, 0.1819, 2.3349, 0.5286])


a_leftS = np.array([a_rightS[2], a_rightS[3], a_rightS[0], a_rightS[1],
                a_rightS[6], a_rightS[7], a_rightS[4], a_rightS[5],
                a_rightS[10], a_rightS[11], a_rightS[8], a_rightS[9],
                a_rightS[14], a_rightS[15], a_rightS[12], a_rightS[13],
                a_rightS[18], a_rightS[19], a_rightS[16], a_rightS[17],
                a_rightS[22], a_rightS[23], a_rightS[20], a_rightS[21]])

p = np.array([0.2517, -0.2])

q = np.array([-0.2000, 0.7546, 0.1675, 2.4948, 0.4405, 3.1894, 0.2132])

